src/App.js

```diff
@@ -4,16 +4,22 @@ import { Routes, Route } from "react-router-dom";
 import { ProductListing } from './pages/product/productListing';
 import { Login } from './pages/login/login';
 import { Signup } from './pages/signUp/signup';
+import { Cart } from './pages/cart/cart';
+import { Navbar } from './component/navbar/navbar';
+import { Footer } from './component/footer/footer';
```

Comment:
You can add an `index.js` file in your pages folder and add all your page exports in that file, then you will be able to import all your pages from a single file.